test-it/karma/utilities/TestHelpers.VireoRunner.js

```diff
@@ -62,11 +62,14 @@
                 });
             }
 
-            var complete = function () {
+            vireo.eggShell.executeSlicesUntilClumpsFinished().then(function () {
+                var noop = function () {
+                    // intentionally blank
```

Comment:
Please add more rationale as to why this section is intentionally blank.

Reply:
Added a comment describing why the print functions were cleared